.github/workflows/auto-pr-merge.yml

```
@@ -111,6 +111,9 @@ jobs:
                 const linkedin_logo = "https://edent.github.io/SuperTinyIcons/images/svg/linkedin.svg";
                 const dev_logo = "https://edent.github.io/SuperTinyIcons/images/svg/dev_to.svg";
                 const hackernews_logo = "https://edent.github.io/SuperTinyIcons/images/svg/hackernews.svg";
+                const bluesky_logo = "https://path-to-your-bluesky-logo.png"; 
```

logo url is wrong. Please use https://edent.github.io/SuperTinyIcons/images/svg/bluesky.svg
```suggestion
                const bluesky_logo = "https://edent.github.io/SuperTinyIcons/images/svg/bluesky.svg"; 
```

---

No worries. The message hasn't been updated. Please check the PR I linked above for reference. 

Also, please remove submodule inside